src/OpenLoco/src/Windows/Options.cpp

```diff
@@ -154,30 +156,48 @@ namespace OpenLoco::Ui::Windows::Options
                 ImageIds::tab_globe_30,
                 ImageIds::tab_globe_31,
             };
-            imageId = ImageIds::tab_globe_0;
-            if (w->currentTab == tab::regional)
+
+            // Regional tab
             {
-                imageId = globe_tab_ids[(w->frameNo / 2) % 32];
+                auto imageId = ImageIds::tab_globe_0;
+                if (w->currentTab == tab::regional)
+                {
+                    imageId = globe_tab_ids[(w->frameNo / 2) % 32];
```

Comment:
i know this is the previous code, but it'd be nice to have constants for the animation frame counts for these tabs

Reply:
It's more or less the result of the options window being one of the first windows we reimplemented. While I agree it would be nice to have constants, I also think such changed would be out of scope for this PR.